src/main/java/at/hannibal2/skyhanni/features/garden/AtmosphericFilterDisplay.kt

```diff
@@ -0,0 +1,71 @@
+package at.hannibal2.skyhanni.features.garden
+
+import at.hannibal2.skyhanni.SkyHanniMod
+import at.hannibal2.skyhanni.events.GuiRenderEvent
+import at.hannibal2.skyhanni.events.LorenzTickEvent
+import at.hannibal2.skyhanni.utils.LorenzUtils
+import at.hannibal2.skyhanni.utils.RenderUtils.renderString
+import at.hannibal2.skyhanni.utils.StringUtils.matchMatcher
+import at.hannibal2.skyhanni.utils.repopatterns.RepoPattern
+import io.github.moulberry.notenoughupdates.util.SkyBlockTime
+import net.minecraftforge.fml.common.eventhandler.SubscribeEvent
+
+class AtmosphericFilterDisplay {
+
+    private val config get() = SkyHanniMod.feature.garden.atmosphericFilterDisplay
+
+    private val atmosphericFilterDisplayGroup = RepoPattern.group("atmosphericfilter.hud")
+    private val seasonPattern by atmosphericFilterDisplayGroup.pattern(
+        "season.skyblocktime",
+        "(?:(?:Early|Late)? )?(?<season>Spring|Summer|Autumn|Winter)"
```

Comment:
this can be simplified to `(?:Early |Late )?(?<season>Spring|Summer|Autumn|Winter)`